testdata/go/obj/api/docs.metadata.newer

```diff
@@ -0,0 +1,7 @@
+update_time {
```

Comment:
Nit: Should this file be under `testdata/go/obj/api` or should it stay under `testdata`? The reason I ask is because I believe this is used for a unit test, but it wouldn't normally be contained in a Go tarball!

Reply:
Good call out. The golden test wasn't right before because it was still calling `docfx` directly, rather than actually going through "doc pipeline." I updated the test and moved these test files to the root (no more `docfx.json` and no more `obj/api`).